### PR TITLE
chore: cleanup docker-compose setup

### PR DESCRIPTION
**Pull request by obdulia-losantos**
There is nothing like reading the docs :nerd_face: 

`./scripts/docker_start.sh` does the trick.

### CODE REVIEW

File: scripts/docker_start.sh [hunk 1]
```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
```
Comment (admbtlr) [inline scripts/docker_start.sh:1]: If I start kernel then odk, `docker-compose` is smart enough to know that kernel is already running, isn't it?

Comment (obdulia-losantos) [reply scripts/docker_start.sh:1]: The next line will kill all running containers first... I can remove/comment that line in that case.

Comment (obdulia-losantos) [reply scripts/docker_start.sh:1]: docker-compose is smart enough to know which containers are running or not and does not restart them again or complain ;-)

Comment (admbtlr) [reply scripts/docker_start.sh:1]: cool. maybe you could hide the `set -e` behind a flag? like maybe `-f` to force killing and restarting the containers?

Comment (obdulia-losantos) [reply scripts/docker_start.sh:1]: better now?

**Comment by obdulia-losantos** [comment]
@MrPink this one fixes all the chaos with the DC (AKA _Detective Comics_) files

File: scripts/docker_start.sh (hunk 1 above)
Comment (admbtlr) [reply scripts/docker_start.sh:1]: beautiful!

**Review by admbtlr — APPROVED** [review]

**Comment by obdulia-losantos** [comment]
😊

**Pull request merged**